EFCache/InMemoryCache.cs

```diff
@@ -165,11 +170,35 @@ public int Count
             get { return _cache.Count; }
         }
 
+        public IEnumerable<string> EntitySetsInCache
+        {
+            get
+            {
+                return _entitySetToKey.Select(x => x.Key);
+            }
+        }
+
+        public int GetCacheSize()

```

Comment:
This method does not return correct value for InMemoryCache - `BinaryFormatter` adds a bunch additional data on top of actual bytes.